zbus/src/address/transport/mod.rs

```diff
@@ -1,6 +1,6 @@
 //! D-Bus transport Information module.
 //!
-//! This module provides the trasport information for D-Bus addresses.
+//! This module provides the transport information for D-Bus addresses.
```

Comment:
thanks for your attention to detail but kindly split separate logical changes into separate commits (not PRs).

Reply:
Lol, you already did this. :facepalm: Sorry for the noise.